pymatviz/process_data.py

```
@@ -448,7 +448,7 @@
 
     if not isinstance(df, pd.DataFrame):
         if not strict:
-            return args  # type: ignore[return-value]
+            return args
         raise TypeError(f"df should be pandas DataFrame or None, got {type(df)}")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)